integrationExamples/gpt/top-level-paapi/decisionLogic.js

```diff
@@ -0,0 +1,44 @@
+const LOG_PREFIX = [
+  '%c PAAPI %c scoreAd %c',
+  'color: green;  background-color:yellow; border: 1px solid black',
+  'color: blue; border:1px solid black',
+  '',
+];
+
+function scoreAd(
+  adMetadata,
+  bid,
+  auctionConfig,
+  trustedScoringSignals,
+  browserSignals,
+  directFromSellerSignals
+) {
+  console.group(...LOG_PREFIX, 'Context:', {
+    adMetadata,
+    bid,
+    auctionConfig,
+    trustedScoringSignals,
+    browserSignals,
+    directFromSellerSignals
+  });
+
+  const result = {
+    desirability: bid,
+    rejectReason: 'not-available',
+    allowComponentAuction: true,
+  };
+  const {bidfloor, bidfloorcur} = auctionConfig.auctionSignals?.prebid || {};
+  if (bidfloor) {
+    if (browserSignals.bidCurrency !== '???' && browserSignals.bidCurrency !== bidfloorcur) {
+      console.log(`Floor currency (${bidfloorcur}) does not match bid currency (${browserSignals.bidCurrency}), and currency conversion is not yet implemented. Rejecting bid.`);
+      result.desirability = -1;
+    } else if (bid < bidfloor) {
+      console.log(`Bid (${bid}) lower than contextual winner/floor (${bidfloor}). Rejecting bid.`);
+      result.desirability = -1;
+      result.rejectReason = 'bid-below-auction-floor';
+    }
+  }
+  console.groupEnd();
+  console.log(...LOG_PREFIX, 'Result:', result);
+  return result;
+}
```

Comment:
`reportResult` should still be defined even if it does not do anything, to avoid
> Worklet error: https://127.0.0.1:9999/integrationExamples/gpt/top-level-paapi/decisionLogic.js `reportResult` is not a function.

Could be something like this
```javascript
function reportResult(auctionConfig, browserSignals) {
  log('reportResult', { auctionConfig, browserSignals });
  sendReportTo(`${auctionConfig.seller}/report/win?${Object.entries(browserSignals).map(([k, v]) => `${k}=${encodeURIComponent(v)}`).join('&')}`);
  return {}
}
```

Reply:
that would replace it with an error about the report call failing :) I am already worried about people copying this demo into production so I'd rather avoid it; the plan is to work with @patmmccann to publish an actual reference implementation separately

Reply:
Is this about the event url showing up as an error in the Network tab ? It's still useful to look at parameters, but if unwanted can comment out sendReportTo or make it conditional on something like sellerSignals.reportURL